### DISCUSSION TITLE
Papoo Installation Xampp php 8.0

### DISCUSSION THREAD

**Discussion by falkei** [PapooSoftware/PapooCMS#11] [Q&A]
Hallo,
leider funktioniert bei mir die aktuelle Papoo-Version in der lokalen Umgebung mit php 8.0 nicht.
Folgender Fehler tritt auf :
fatal error uncaught error undefined constant php 8.....
Kann mir jemand dazu etwas sagen?
Gruß Ina

**Comment by ah-papoo** [comment]
Bitte geben Sie die vollständige Fehlermeldung an und beschreiben Sie, wann diese auftritt.

**Reply by falkei** [reply]
Tut mir leid, ich hatte die falsche Papoo-Version am Start. Mit der neuesten Version läuft es!
Danke trotzdem für die Antwort.

**Reply by falkei** [reply]
Jetzt allerdings doch noch eine weitere Frage.
Sobald das Content Manipulator Plugin installiert ist, kommt folgender Fehler:
Fatal error: Uncaught TypeError: mktime(): Argument #6 ($year) must be of type .....
Wird das Plugin deinstalliert, läuft die Seite.
Gruß Ina

**Reply by ah-papoo** [reply]
Wenn Sie nicht die vollständige Fehlermeldung posten, hat es keinen Sinn zu versuchen Ihnen zu helfen.
Der Fehler lässt aber drauf schließen, dass PHP 7.4 zu verwenden das Problem lösen wurde, soweit ich mich erinnere, wurde das Problem auch bereits behoben, aber noch keine Release dafür erstellt. Verwenden Sie einfach die aktuelle Dev-Version:
![Bildschirmfoto vom 2022-08-25 17-33-07](https://user-images.githubusercontent.com/69973945/186708136-22b4f85b-a827-4850-ae4b-a90d93df199f.png)

Bitte bei weiteren Problemen die **vollständige** Fehlermeldung dazusenden.

**Comment by falkei** [comment]
Guten Morgen,
ich würde gerne php8 verwenden, da im Forum auch gesagt wurde, dass es mit
dieser Version funktioniert.
Es läuft alles soweit, bis ich das Plugin Content Manipulator installiere.

Meine jetzige Installation sieht so aus:
Ihre Version ist: *21.05 Rev. de2d1cd - Papoo Community*
PHP Version 8.0.19

Die Fehlermeldung lautet wie folgt:
*Fatal error*: Uncaught TypeError: mktime(): Argument #6 ($year) must be of
type ?int, string given in
C:\xampp\htdocs\papoo\lib\classes\artikel_class.php:1949 Stack trace: #0
C:\xampp\htdocs\papoo\lib\classes\artikel_class.php(1949): mktime('23',
'00', 0, '12', '09', '2021 23:00') #1
C:\xampp\htdocs\papoo\lib\classes\artikel_class.php(2579):
artikel_class->get_artikel('20', '', 0) #2
C:\xampp\htdocs\papoo\plugins\content_manipulator\scripts\letzte_artikel\include.php(210):
artikel_class->make_teaser() #3
C:\xampp\htdocs\papoo\plugins\content_manipulator\scripts\letzte_artikel\include.php(71):
last_artikel->get_last_artikel('4', '20', '1') #4
C:\xampp\htdocs\papoo\plugins\content_manipulator\scripts\letzte_artikel\include.php(35):
last_artikel->create_last_artikelintegration('<!doctype html>...') #5
C:\xampp\htdocs\papoo\plugins\content_manipulator\scripts\letzte_artikel\include.php(451):
last_artikel->__construct() #6
C:\xampp\htdocs\papoo\plugins\content_manipulator\lib\content_manipulator.php(65):
require_once('C:\\xampp\\htdocs...') #7
C:\xampp\htdocs\papoo\plugins\content_manipulator\lib\content_manipulator.php(42):
content_manipulator->filter_ausgabe_inhalt() #8
C:\xampp\htdocs\papoo\lib\classes\pluginintegrator_class.php(195):
content_manipulator->output_filter() #9
C:\xampp\htdocs\papoo\lib\settings\output_pars.php(5):
pluginintegrator_class->output_filter() #10
C:\xampp\htdocs\papoo\all_inc_front.php(165):
require_once('C:\\xampp\\htdocs...') #11
C:\xampp\htdocs\papoo\index.php(17): require_once('C:\\xampp\\htdocs...')
#12 {main} thrown in *C:\xampp\htdocs\papoo\lib\classes\artikel_class.php*
on line *1949*

Gruß Ina Falke

Am Do., 25. Aug. 2022 um 17:35 Uhr schrieb ah-papoo <
***@***.***>:

**Reply by ah-papoo** [reply]
Der Fehler wurde bereits behoben: https://github.com/PapooSoftware/PapooCMS/commit/2a292311fbeb69eee3e278b186140fc222949e4f
Wie ich schon sagte, einfach die Dev-Version nutzen.

**Comment by falkei** [comment]
Danke, mir war nicht klar, dass ein Unterschied besteht zwischen dem
Download der Version 21.12.14 und der Dev-Version (was immer das auch ist).
Jetzt läuft es jedenfalls.


Am Fr., 26. Aug. 2022 um 10:48 Uhr schrieb ah-papoo <
***@***.***>:
